stdpopsim/catalog/MusMus/demographic_models.py

```diff
@@ -0,0 +1,344 @@
+            1443,
+            1754,
+            2114,
+            2530,
+            3012,
+            3570,
+            4216,
+            4964,
+            5829,
+            6831,
+            7991,
+            9334,
+            10889,
+            12688,
+            14772,
+            17183,
+            19975,
+            23207,
+            26948,
+            31279,
+            36292,
+            42096,
+            48815,
+            56592,
+            65596,
+            76019,
+            88084,
+            102052,
+            118221,
+            136938,
+            158606,
+            183689,
+            212726,
+            246340,
+            285254,
+            330300,
+            382446,
+            442812,
+            512693,
+            593589,
+            687237,
+            795646,
+            921140,
+            1066418,
+            1234595,
+            1429281,
+            1654653,
+            1915544,
+        ]
+    )
+    sizes = np.array(
+        [
+            2040,
+            2040,
+            3844,
+            90428,
+            145603,
+            111242,
+            115399,
+            147212,
+            159142,
+            136620,
+            97250,
+            58488,
+            33028,
+            18939,
+            11758,
+            8463,
+            7480,
+            8332,
+            11240,
+            16490,
+            23419,
+            29931,
+            34163,
+            36886,
+            41195,
+            50557,
+            67337,
+            90926,
+            115426,
+            131016,
+            132063,
+            121751,
+            107067,
+            93046,
+            81892,
+            74185,
+            69939,
+            69317,
+            73097,
+            82953,
+            101471,
+            131392,
+            173264,
+            222951,
+            271935,
+            309961,
+            327217,
+            316861,
+            279833,
+            227037,
+            173594,
+            131050,
+            98811,
+            98811,
+            133912,
+            133912,
+            133912,
+        ]
+    )
+
+    demographic_events = []
+    for sz, t in zip(sizes, times):
+        demographic_events.append(
+            msprime.PopulationParametersChange(time=t, initial_size=sz, population_id=0)
+        )
+
+    populations = [
+        stdpopsim.Population(
+            id="M_musculus_domesticus",
+            description="Mus musculus domesticus German population",
+        )
+    ]
+
+    return stdpopsim.DemographicModel(
+        id="M_musculus_domesticus_Europe",
+        description="M. musculus domesticus piecewise constant size",
+        long_description="""
+            This model comes from MSMC using four randomly sampled
+            individuals (DEU01,DEU03,DEU04,DEU06) from a German population.
+            The model is estimated with 57 time periods.
+        """,
```

Comment:
I would also explicitly mention Fig 3 of Fujiwara et al 2022 and say that the population sizes and time changes were supplied by the authors (maybe even mention a specific author?).

Reply:
I have added these in.

Reply:
Great